packages/webapp/src/components/Task/TaskComplete/HarvestComplete/HarvestUses.js

```diff
@@ -108,7 +113,7 @@ export default function PureHarvestUses({
 
       <Label style={{ marginBottom: '16px' }}>
         {t('TASK.AMOUNT_TO_ALLOCATE')}
-        <UnitLabel style={{ marginLeft: '12px' }} amount={amount_to_allocate} unitLabel={unit} />
+        <UnitLabel style={{ marginLeft: '12px' }} amount={amount_to_allocate} unitLabel={display_unit[system]} />
```

Comment:
Why is this change necessary? It seems to me that `unit` is still being passed in as a prop with the value `persistedFormData?.actual_quantity_unit?.label`. Is that not being used?